src/harfbuzz-config.cmake.in

```diff
@@ -0,0 +1,5 @@
+add_library(harfbuzz::harfbuzz UNKNOWN IMPORTED)
+set_target_properties(harfbuzz::harfbuzz PROPERTIES
+  INTERFACE_INCLUDE_DIRECTORIES "@prefix@/include/harfbuzz"
+  IMPORTED_LOCATION "@prefix@/lib/harfbuzz"
```

Comment:
This should be the path to the actual library, so `lib` -> `@libdir@` and if autoconf has access to the name of the library installed, that should be used, otherwise some logic will be required.

Reply:
```
# libdir = @libdir@
# exec_prefix = @exec_prefix@
# prefix = @prefix@
```
will be turned to:

```
# libdir = ${exec_prefix}/lib
# exec_prefix = ${prefix}
# prefix = /usr/local
```

can cmake it self resolves that out somehow?

Reply:
Hmm, it should be able, let me test